hooks/command

```diff
@@ -8,12 +8,26 @@ if [[ "${BUILDKITE_PLUGIN_DOCKER_DEBUG:-false}" =~ (true|on|1) ]] ; then
   debug_mode='on'
 fi
 
-args=(
-  "-it"
-  "--rm"
-  "--volume" "$PWD:${BUILDKITE_PLUGIN_DOCKER_WORKDIR:-/workdir}"
-  "--workdir" "${BUILDKITE_PLUGIN_DOCKER_WORKDIR:-/workdir}"
-)
+windows_image () { [[ "${BUILDKITE_PLUGIN_DOCKER_WINDOWS_IMAGE:-false}" =~ (true|on|1) ]]; }
+
+args=("--rm")
+
+if ! windows_image; then
+  args+=("-it")
```

Comment:
`it` didn't work when the plugin was used on a Windows host. Honestly, I wonder if we can just get rid of `-it` altogether. How is it useful for linux build jobs?